cypress/integration/v2/features/global-settings/global-typography.spec.js

```diff
@@ -1,13 +1,13 @@
 /**
  * External dependencies
  */
+
 import { range } from 'lodash'
 import { registerBlockSnapshots } from '~gutenberg-e2e/plugins'
 import {
 	responsiveAssertHelper,
 	registerTests,
 } from '~stackable-e2e/helpers'
-
 import { blocks } from '~stackable-e2e/config'
 
 const [ , tabletGlobalTypo, mobileGlobalTypo ] = responsiveAssertHelper( assertGlobalTypographyTabletMobile, { disableItAssertion: true } )
```

Comment:
Let's add desktopGlobalTypography here.

Can we also change the function names? Use `typography` instead of `typo`